pkg/config/template.go

```diff
@@ -85,6 +88,22 @@ func NewTemplateCMManager(cmclient corev1.ConfigMapInterface, prNumber int, logg
 	}
 }
 
+func (c *TemplateCMManager) createCM(cm *v1.ConfigMap) error {
+	l := &cm.ObjectMeta.Labels
+	if *l == nil {
+		*l = map[string]string{}
+	}
+	(*l)[createByRehearse] = "true"
```

Comment:
Why can't we:

```go
if cm.ObjectMeta.Labels == nil {
    cm.ObjectMeta.Labels = map[string]string{}
}
cm.ObjectMeta.Labels[createByRehearse] = "true"
cm.ObjectMeta.Labels[rehearseLabelPull] = strconv.Itoa(c.prNumber)
```

Reply:
I wrote the exact same code as a comparison when discussing this with @droslean.

Personal preference, I guess, but it's 2x1 now. Changed.